deepchem/molnet/run_benchmark.py

```diff
@@ -22,6 +22,7 @@ def run_benchmark(datasets,
                   split=None,
```

Comment:
Suggest you to do the loop outside the function.
Here we only accept multiple datasets for convenience, if multiple splits, models are all permitted, things could go too complex.